kairon/shared/channels/mail/processor.py

```
@@ -1,14 +1,18 @@
 from loguru import logger
 from pydantic.schema import timedelta
 from pydantic.validators import datetime
-from imap_tools import MailBox, AND
+from imap_tools import MailBox, AND, OR, NOT
+
+from kairon import Utility
 from kairon.exceptions import AppException
 from kairon.shared.account.data_objects import Bot
 from kairon.shared.channels.mail.constants import MailConstants
 from kairon.shared.channels.mail.data_objects import MailResponseLog, MailStatus, MailChannelStateData
+from kairon.shared.chat.data_objects import Channels
```

Ruff (0.8.2)

`kairon.shared.chat.data_objects.Channels` imported but unused

Remove unused import: `kairon.shared.chat.data_objects.Channels`

(F401)